sympy/parsing/mathematica.py

```diff
@@ -58,7 +58,11 @@ class MathematicaParser:
         'LogIntegral[x]':' li(x)',
         'PrimePi[x]': 'primepi(x)',
         'Prime[x]': 'prime(x)',
-        'PrimeQ[x]': 'isprime(x)'
+        'PrimeQ[x]': 'isprime(x)',
+        'Factorial[x]': 'factorial(x)',
+        'FactorialPower[x,y]': 'ff(x,y)',
+        'Binomial[x,y]': 'binomial(x,y,True)',
```

Comment:
[Mathematica](https://www.wolframalpha.com/input/?i=calculatorbinomial%28-3%2C-4%29) uses the generalized definition: `binomial(-3,-4)` is not 0, it is -3.